READMEs/data-nfts-and-datatokens-flow.md

```diff
@@ -52,6 +52,19 @@ pip3 install wheel
 pip3 install --pre ocean-lib
 ```
 
+For M1 processors, if the installation of ocean lib fails due to `coincurve` or
```

Comment:
I would create a subsection or a different readme called "Known issues" and include a shortened version of this content. How about that? @trentmc @mariacarmina